### PR TITLE
[PWGLF] Add centrality info, fix minor issues

### PR DESCRIPTION
**Pull request by mhartung71**

### CODE REVIEW

**Review by njacazio — APPROVED** [review]

**Comment by mhartung71** [comment]
Hi @njacazio, the confilcts are now resolved

**Review by njacazio — APPROVED** [review]

File: PWGLF/TableProducer/Nuspex/hypKfRecoTask.cxx [hunk 1]
```diff
@@ -403,7 +396,7 @@ std::vector<std::shared_ptr<TH1>> hInvMass;
 
 //----------------------------------------------------------------------------------------------------------------
 //----------------------------------------------------------------------------------------------------------------
-struct hypKfRecoTask {
+struct hypKfRecoTask { // o2-linter: disable=[name/workflow-file][name/struct]
```
Comment (vkucera) [inline PWGLF/TableProducer/Nuspex/hypKfRecoTask.cxx:399]: What is the reason to disable the tests here?

File: PWGLF/TableProducer/Nuspex/hypKfRecoTask.cxx [hunk 2]
```diff
@@ -369,32 +348,46 @@ struct hyperNucCandidate {
     subDaughter.TransportToDecayVertex();
     return subDaughter.GetMass();
   }
-}; // class hyperNucCandidate
+  KFParticle getDaughterTrackKfp(int track)
+  {
+    return kfpDaughters.at(track + isCascade() ? 1 : 0);
```
Comment (vkucera) [inline PWGLF/TableProducer/Nuspex/hypKfRecoTask.cxx:353]: This is a bug reported as a compilation error. Please fix it asap.

**Review by vkucera — COMMENTED** [review]
@mhartung71 Thanks a lot for addressing the O2 linter warnings. Please see my comments below.

**Comment by mhartung71** [comment]
Dear Vit,

I have just committed a bug fix.

Best regards, Michael

Am Mittwoch, dem 18.12.2024 um 11:16 -0800 schrieb Vít Kučera:

**Comment by vkucera** [comment]
Thanks @mhartung71

**Pull request merged**